metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/Jobs.scala

```diff
@@ -36,6 +36,8 @@ class Jobs(config: Configuration, readModel: ReadModelService, s3: S3)(implicit
             a.eservice,
             a.producer,
             a.consumer,
+            a.consumerId,
+            a.producerId,
```

Comment:
these should follow the order of the header
```suggestion
            a.producerId,
            a.consumer,
            a.consumerId,
```